packages/instrumentor/plugins/helpers.ts

```diff
@@ -2,6 +2,10 @@ import { NumericLiteral } from "@babel/types";
 import { types } from "@babel/core";
 import * as crypto from "crypto";
 
+export function nextFakePC(): number {
```

Comment:
Is this change / the export necessary, as `nextFakePC` is not used outside this file.

Reply:
reverted to the version before the bug detectors.